clang/include/clang/Driver/Options.td

```diff
@@ -7730,6 +7730,14 @@ def source_date_epoch : Separate<["-"], "source-date-epoch">,
 
 } // let Visibility = [CC1Option]
 
+def stdlib_hardening_EQ : Joined<["-"], "fstdlib-hardening=">,
```

Comment:
With the name `stdlib_hardening`, are we worried that users might think it applies to the C standard library as well? Should the name imply that this is only for C++?

Reply:
I am worried about that -- I'm also worried about folks thinking it applies to more than libc++.

Reply:
It is consistent with `-stdlib`